cgroups/fs/apply_raw.go

```diff
@@ -58,7 +58,9 @@ type subsystem interface {
 	// Removes the cgroup represented by 'data'.
 	Remove(*data) error
 	// Creates and joins the cgroup represented by data.
-	Set(*data) error
+	Apply(*data) error
+	// Set the cgroup represented by cgroup
+	Set(path string, cgroup *cgroups.Cgroup) error
```

Comment:
Because this could be used for the first time you `set` the resource configs (called from `Apply`), and the API name is also inspired from Solomon in a previous PR https://github.com/docker/docker/pull/3604, that's what I also plan to use on Docker side.

Anyway I'm not insisted on the name, I'll change it if there are more people think we have a better choice.